pages/platform/databases/connect-kubernetes-to-managed-mysql/guide.en-asia.md

```diff
@@ -0,0 +1,302 @@
+NAME: my-wordpress
+LAST DEPLOYED: Thu Dec 23 15:49:33 2021
+NAMESPACE: default
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+NOTES:
+CHART NAME: wordpress
+CHART VERSION: 12.2.7
+APP VERSION: 5.8.2
+
+** Please be patient while the chart is being deployed **
+
+Your WordPress site can be accessed through the following DNS name from within your cluster:
+
+    my-wordpress.default.svc.cluster.local (port 80)
+
+To access your WordPress site from outside the cluster follow the steps below:
+
+1. Get the WordPress URL by running these commands:
+
+  NOTE: It may take a few minutes for the LoadBalancer IP to be available.
+        Watch the status with: 'kubectl get svc --namespace default -w my-wordpress'
+
+   export SERVICE_IP=$(kubectl get svc --namespace default my-wordpress --template "{{ range (index .status.loadBalancer.ingress 0) }}{{.}}{{ end }}")
+   echo "WordPress URL: http://$SERVICE_IP/"
+   echo "WordPress Admin URL: http://$SERVICE_IP/admin"
+
+2. Open a browser and access WordPress using the obtained URL.  
+
+3. Login with the following credentials below to see your blog:
+
+  echo Username: user
+  echo Password: $(kubectl get secret --namespace default my-wordpress -o jsonpath="{.data.wordpress-password}" | base64 --decode)
+
+</code></pre>
+
+> [!Warning]
+> Be sure your MySQL database defaultdb is clean before executing the helm install command. If a previous installation is detected, parameters like user and password won't be updated, so the k8s configuration will mismatch with the database one.
+
+As the instructions say, you will need to wait a few moments to get the `LoadBalancer` URL.
+You can test if the `LoadBalancer` is ready using:
+
+```
+kubectl get svc --namespace default -w my-wordpress
+```
+
+After some minutes, you will get the `LoadBalancer` URL:
+
+<pre class="console"><code>$ kubectl get svc --namespace default -w my-wordpress
+NAME           TYPE           CLUSTER-IP     EXTERNAL-IP      PORT(S)                      AGE
+my-wordpress   LoadBalancer   10.3.193.143   135.125.83.116   80:32027/TCP,443:32293/TCP   4m30s
+</code></pre>
+
+Then you can follow the instructions to get the Admin URL:
+
+<pre class="console"><code>$ export SERVICE_IP=$(kubectl get svc --namespace default my-wordpress --template "{{ range (index .status.loadBalancer.ingress 0) }}{{.}}{{ end }}")
+$ echo "WordPress URL: http://$SERVICE_IP/"
+WordPress URL: http://135.125.83.116/
+$ echo "WordPress Admin URL: http://$SERVICE_IP/admin"
+WordPress Admin URL: http://135.125.83.116/admin
+</code></pre>
+
+And putting the URL in your browser will take you to the new blog:
+
+![Installing Wordpress](images/connect-kubernetes-to-managed-mysql12.png){.thumbnail}
+
+You also use the instructions given by the helm install to get the default username and password for your blog.
```

Comment:
"You can also use the instructions given by the` helm install`command to get the default username and password for your blog.
"